apps/backend/src/modules/activity-logs/activity-log.service.ts

```diff
@@ -136,6 +139,60 @@ class ActivityLogService implements Service {
 		};
 	}
 
+	public async findAllWithParameters({
```

Comment:
let's name just findAll
```suggestion
	public async findAll({
```